Makefile

```diff
@@ -283,8 +283,13 @@ ${SHFMT}:
 ${GO_MOD_OUTDATED}:
 	$(call go-build,./vendor/github.com/psampaz/go-mod-outdated)
 
+ZEITGEIST_VERSION := v0.3.5
```

Comment:
I recommend moving this change to a separate commit so that the vendor changes are in a specific commit.

Reply:
Yep, done.